common/config/config.go

```diff
@@ -150,6 +150,11 @@ type Config struct {
 	MultiSync struct {
 		// Enabled bool `envconfig:"STARHUB_SERVER_MULTI_SYNC_ENABLED" default:"false"`
 	}
+
+	Telemetry struct {
+		Enable    bool   `envconfig:"STARHUB_SERVER_TELEMETRY_ENABLE" default:"true"`
+		ReportURL string `envconfig:"STARHUB_SERVER_TELEMETRY_URL" default:"http://hub.opencsg.com/api/v1/telemetry"`
```

Comment:
API was not implemented

Reply:
see this PR: https://github.com/OpenCSGs/csghub-server/pull/78